tox.ini

```diff
@@ -9,6 +9,14 @@ deps =
 
 commands = pylint -rn --rcfile={toxinidir}/pylintrc --load-plugins=pylint.extensions.docparams, pylint.extensions.mccabe {envsitepackagesdir}/pylint
 
+[testenv:mypy]
+basepython = python3
+deps =
+    mypy
+
+commands =
+    python -m mypy {envsitepackagesdir}/pylint/checkers --ignore-missing-imports
```

Comment:
Maybe run it against the current directory instead of the package installed in `site-packages`?